dev-system/dev/user.clj

```diff
@@ -117,6 +117,20 @@
   []
   (set-modes! :all settings/default-run-mode))
 
+(defn- distribute-schemas
+  "A function that will copy the directory OtherSchemas in to all the necessary
+   apps and will be called on reset"
+  []
+  (let [path (System/getProperty "user.dir")
+        apps ["search-app"
+              "ingest-app"
+              "indexer-app"
+              "metadata-db-app"
```

Comment:
metadata db is needed?

Reply:
It was part of the acceptance criteria in the ticket.

Reply:
also these all go inside a directory like indexer-app/resources/schemas/